source/_posts/2021-06-09-home-assistant-os-release-6.markdown

```diff
@@ -0,0 +1,145 @@
+
+- The short name of the OS used throughout the software stack was renamed to
+  "haos". From a users perspective not much changes, but it leads to new file
+  names.
+- The Home Assistant CLI is now started on tty1 instead of the login prompt. The
+  HA CLI banner shows the IP and other device information without the need
+  to attach a keyboard to login.
+- The default shell of the root user is now the Busybox ash. This allows to use
+  scp and similar tools using SSH.
+- The system start is now delayed until the system time has been synchronized
+  with an NTP server. This avoids connection issues to SSL/TLS enabled servers
+  when the system time is off. In practice, the delay should only be a few
+  seconds. The system proceeds with startup after 90s in case the NTP server was
+  not reachable (e.g. if no Internet connectivity is available).
+
+## Device Support
+
+### Raspberry Pi
+
+All Raspberry Pi versions were updated to use the Linux Kernel 5.10 from the
+Raspberry Pi team, just like Raspberry Pi OS. The latest Bluetooth firmware for
+CYW43455 (Raspberry Pi 3 B+/4) with security fixes is now part of the Raspberry
+Pi images.
+
+### Generic x86-64
+
+In Home Assistant OS 6 the board/image was renamed from "Intel NUC" to "Generic
+x86-64". Existing "Intel NUC" installation will automatically migrate to the
+"Generic x86-64" image on update.
+
+Already with OS 5 the "Intel NUC" image was usable for most UEFI capable x86-64
+systems. With the rename that fact is emphasized. Home Assistant OS is open
+to changes to add drivers to extend support for other x86-64 platforms. However,
+keep in mind that we don't have access to all x86-64 hardware, so support for
+devices is on a best effort basis.
+
+Other Changes:
+
+- The Intel IGC network driver supports Intel network cards such as I225-LM/I225-V
+  used on some newer Intel NUC devices.
+
+- With Home Assistant OS 6 the latest version of the Barebox bootloader 2021.05.0
+  which fixes boot from hard disks larger than 2TB.
+
+- The ACPI kernel driver for Thinkpad devices is enabled.
+
+### Open Virtualization Appliance
+
+The virtual disk images are now built from a sparse file as source, which leads
+to smaller effective disk image sizes. Disk images are now also distributed in
+the more common zip file format (except the qcow2 images which are mostly used on
+Linux, they continue to be in the xz compression format).
+
+By default, the image now has a serial console enabled which is useful to access
+Home Assistant OS on headless host systems.
+
+### Additional WiFi and Bluetooth Device Support
+
+Ralink rt27xx/rt28xx/rt30xx as well as Realtek 87xx/88xx devices are now
+supported on boards which don't have on-board WiFi/Bluetooth. This is also
+true for the OVA image for virtualized environments (when using hardware
+passthrough functionality). For the OVA image, also Intel firmwares are now
+part of the image so that Intel WiFi/Bluetooth devices can be used in
+passthrough mode as well.
```

Comment:
```suggestion
Ralink rt27xx/rt28xx/rt30xx, as well as Realtek 87xx/88xx devices, are now
supported on boards that don't have onboard Wi-Fi/Bluetooth. This is also
true for the OVA image for virtualized environments (when using hardware
passthrough functionality). For the OVA image, also Intel firmwares are now
part of the image so that Intel WiFi/Bluetooth devices can be used in
passthrough mode as well.
```